examples/.gitignore

```diff
@@ -0,0 +1,2 @@
+Toolchain
+nuget.config
```

Comment:
Why do you need a nuget.config ?

Reply:
Because you can install the toolset right into this folder which creates a nuget.config